src/__tests__/app.integration.test.js

```diff
@@ -28,23 +48,30 @@ describe('GET /health', () => {
       }
     };
 
-    const res = await request(app).get(`/health`);
+    const response = request(app)
```

Comment:
an `await` before request(app) is missing in line 51
before change it was ```const res = await request(app).get(`/health\`);```

Reply:
Good spot, although this doesn't seem to be part of 3162

Reply:
I agree! Though this is outside the scope of 3162 I would argue, this was implemented in a previous ticket. Fix added nonetheless.